Jenkinsfile

```diff
@@ -1,369 +0,0 @@
-def notify(status){
```

Comment:
Be sure to make a back up :)

Reply:
It's still in the master-backup branch (and commit history)